DESCRIPTION

```diff
@@ -1,5 +1,5 @@
 Package: rix
-Title: Rix: Reproducible Environments with Nix
+Title: Reproducible Environments with 'Nix'
```

Comment:
```suggestion
Title: Reproducible Software Environments with 'Nix'
```

Reply:
Could/should we go with "Development" instead of "Software"?

Reply:
yeah; or "Data Science"? for me it can be both dev, prod, deplyoment related topics Nix can address

Reply:
"Data Science" sounds general enough :)